### DISCUSSION TITLE
Codespace doesn't load

### DISCUSSION THREAD

**Discussion by ghost** [community/community#122175] [Codespaces]
### Body

My GH codespace doesn't load, jus says: setting up your codespace. Please help me with this problem.
![Screenshot_1](https://github.com/community/community/assets/168844659/44710fd1-8900-4cb7-a754-9c840d0ca705)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by vinihsr** [comment]
Check your internet connetion, clear your browser chache, try a different browser, check your browser extensions or check for any network restrictions...

**Reply by iasdomingos03out** [reply]
Hello, I am with the same issue. Here, he try load but not occours. Already tried clear cache, change the browser, open in a tab anonymous. My network are good and not have extensions. If i create a new codespace he too not worked and loaded. Already westerday was (05/06/24) working.
![image](https://github.com/community/community/assets/166149821/3ffa890d-ca70-4bd2-9345-f78625870f43)

After a time, its occours.
![image](https://github.com/community/community/assets/166149821/0324f7ea-75cb-42a8-99dd-aac84616498d)
But even trying to reload, it doesn't work.

Could you please help me?

**Reply by rongxin-liu** [reply]
Could you please do the following steps and let me know if the Connection V2 feature is on:

1. Visit [github.com](http://github.com/) and click on your profile image on the top right
2. Click "Feature Preview"

Was the "Codespaces Connectivity v2" feature enabled or disabled? Please send us a screenshot.

**Reply by iasdomingos03out** [reply]
Hi. I think that is a intermittence. But will go follow the steps. Actually this feature are disabled. What is that? 
Thanks for the help Mr. Rongxin-liu!
![image](https://github.com/community/community/assets/166149821/d9802944-3f46-4883-a813-fb1467d7b9cd)

**Reply by 54145a** [reply]
Same issue here 😭

**Comment by rongxin-liu** [comment]
> ### Body
> My GH codespace doesn't load, jus says: setting up your codespace. Please help me with this problem. ![Screenshot_1](https://private-user-images.githubusercontent.com/168844659/327797227-44710fd1-8900-4cb7-a754-9c840d0ca705.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU0MDA3MjcsIm5iZiI6MTcxNTQwMDQyNywicGF0aCI6Ii8xNjg4NDQ2NTkvMzI3Nzk3MjI3LTQ0NzEwZmQxLTg5MDAtNGNiNy1hNzU0LTljODQwZDBjYTcwNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMVQwNDA3MDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMTg1MGQ0N2I3MDZhZTVmNWMyMGEyNzg5YjU5MWIxYWY0YTI4MjBjOWE5ZDdjNTA5YWI5OGY3YWViOGZkMGIwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.XiufzejepcEgPsvZR4kztJ6274y3xraHNrKg0xc1BVQ)
> 
> ### Guidelines
> * [x]  I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

Please try doing the following steps:

1. Assuming using Chrome browser, click "View" -> "Developer" -> "Developer Tools"
2. Select the "Network" tab, and check "Disable cache" and "Preserve Log"
3. Try accessing your codespace and wait for it to load for 1-2 minutes
4. Then, right-click anywhere under the "Network" tab and select "Save all as HAR with content" to save the HAR file to your computer
5. Click the "Console" tab in your browser dev tool and right-click anywhere to download the log file
6. Send us the HAR file and the console log file

**Reply by rongxin-liu** [reply]
We have seen multiple reports like this, and usually, the issue is caused by a local ISP blocking some GitHub-related domains.

**Comment by jojomba** [comment]
I tried to embed the code with my social grant page but it is not working at all. Any one can tell me about how can i embed a code in my page my page is [Grant Payment](https://srdstatuschecks.co.za/srd-r350-grant-at-boxer/).

**Comment by shanrath** [comment]
Firefox enhanced tracking needed to be disabled for me to have working codespace connections:

![image](https://github.com/community/community/assets/7525781/862312b3-8fb7-499c-a354-e7bc6a658e2c)

I'm on version 127.02 (64-bit)
![image](https://github.com/community/community/assets/7525781/4d6e2011-2606-418f-b946-dc64e0b18082)
Needed to disable this on the specific codespaces domain. Already 'reported broken site' to firefox.

**Reply by shanrath** [reply]
1 caveat: i also disabled the uBlock extension, but i think it still works with it enabled afterwards.

**Reply by godlykmathews** [reply]
Thankyou so much

**Reply by edm00se** [reply]
Thank you. This is what was causing the issue for me as well.

**Reply by leftweb** [reply]
`<legend />`

**Comment by 54145a** [comment]
Seems like cors problem(?)
[literate-guide-ggpqw5vqgg9hp4v5.github.dev-1722569655133.log](https://github.com/user-attachments/files/16464999/literate-guide-ggpqw5vqgg9hp4v5.github.dev-1722569655133.log)
